### PR TITLE
Added Japanese Translation of 3 md files (japanese review requested)

### PR DESCRIPTION
**Pull request by aneeshd27**
## What does this PR do?
 Improve repo by adding Japanese Translation to md files

### Why is this valuable (or not)?
A lot of users are interested in Japanese Translation due to adding these files will help in reading and contributing to the repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
language code for japanese is `ja` not `jp` - change the filenames
add links to README
will ask for review

**Comment by aneeshd27** [comment]
I have changed the names of jp named files to ja. Also added links to the files in the docs folder in the README.md file

File: docs/CODE_OF_CONDUCT-ja.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# 貢献者の行動規範
+
+このプロジェクトの貢献者および管理者として、また、開かれた歓迎のコミュニティを育むために
+オープンで歓迎されるコミュニティを育成するために、私たちは、問題の報告、機能追加の要望の投稿、更新を通じて貢献するすべての人々を尊重することを誓います。
+を尊重することを誓います。
```
Comment (whitehara) [inline docs/CODE_OF_CONDUCT-ja.md:5]: This line is not needed.

File: docs/CODE_OF_CONDUCT-ja.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+* 公的または私的な嫌がらせ
+* 明示的な許可なく、物理的または電子的な住所など、他人の個人情報を公開すること。
+  明示的な許可なく、他人の住所を公開すること。
+* その他の非倫理的または非プロフェッショナルな行為
+
+プロジェクトメンテナには、以下を削除、編集、拒否する権利と責任があります。
+コメント、コミット、コード、ウィキ編集、課題、その他の貢献を拒否する権利があります。
+この行動規範に沿わない投稿を削除、編集、拒否する権利があります。
+また、不適切と判断されるその他の行為について、投稿者を一時的または恒久的に追放することがあります、
+を一時的または恒久的に禁止することができます。
```
Comment (whitehara) [inline docs/CODE_OF_CONDUCT-ja.md:27]: This line is not needed.

File: docs/CODE_OF_CONDUCT-ja.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+この行動規範を採用することで、プロジェクトのメンテナーは以下のことを約束します。
+この行動規範を採用することによって、プロジェクトメンテナはこの原則をこのプロジェク
+することを約束するものとします。この行動規範に従わない、あるいは強制しないプロジェクトメンテナは、プロジェクトメンテナから永久に解任されることがあります。
+行動規範に従わないプロジェクトメンテナは、プロジェクトチームから永久に排除される可能性があります。
+
```
Comment (whitehara) [inline docs/CODE_OF_CONDUCT-ja.md:33]: From line 29 to 32, it looks like the same things are explained double.

File: docs/CODE_OF_CONDUCT-ja.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+この行動規範は、プロジェクト空間内および公共の場の両方に適用されます。
+個人がプロジェクトやそのコミュニティを代表しているときに適用されます。
+
+虐待、ハラスメント、その他容認できない行為があった場合は、以下の連絡先まで報告してください。
+プロジェクトメンテナ（victorfelder at gmail.com）までご連絡ください。すべての
+すべての苦情は検討され、調査され、その結果、状況に応じて必要かつ適切とみなされる対応がなされます。
+が必要かつ適切であるとみなされます。メンテナーは
+報告者の秘密を守る義務があります。
+を守らなければならない。
+
```
Comment (whitehara) [inline docs/CODE_OF_CONDUCT-ja.md:43]: From line 37 to 42, it looks like the same things are explained double.

**Comment by eshellman** [comment]
Thank you @whitehara for review.

File: docs/CODE_OF_CONDUCT-ja.md (hunk 2 above)
Comment (aneeshd27) [reply docs/CODE_OF_CONDUCT-ja.md:25]: I have made the necessary changes

File: docs/CODE_OF_CONDUCT-ja.md (hunk 4 above)
Comment (aneeshd27) [reply docs/CODE_OF_CONDUCT-ja.md:40]: Also made changes to it

File: docs/CONTRIBUTING-jp.md [hunk 5]
```diff
@@ -0,0 +1,271 @@
...
+- 適切な場合には、著者名を含めてください。著者リストは"`et al.`"で短縮できます。
+- 本が完成しておらず、まだ作業中である場合は、[下記](#in_process)のように「`in process`」の表記を加えてください。
+- リソースが[*Internet Archive's Wayback Machine*](https://web.archive.org)(または同様のもの)を使って復元された場合は、[下記](#archived)のように"`archived`"表記を追加してください。使用するのに最適なバージョンは、最近の完全なものです。
+- ダウンロードを有効にする前に、メールアドレスやアカウントの設定が必要な場合は、括弧の中に言語に適した注釈を追加してください： メールアドレスは必須ではありません。
+
+
+### フォーマット
+
+- すべてのリストは `.md` ファイルです。Markdown](https://guides.github.com/features/mastering-markdown/)の構文を覚えてください。簡単です！
+- すべてのリストはインデックスから始まります。そこですべてのセクションとサブセクションをリストアップし、リンクすることです。アルファベット順にしてください。
```
Comment (whitehara) [inline docs/CONTRIBUTING-jp.md]: "[" is missed before "Markdown".

File: docs/CONTRIBUTING-jp.md [hunk 6]
```diff
@@ -0,0 +1,271 @@
...
+3. リストには6種類あります。適切なものを選んでください：
+
+    - 書籍* ： 本* : PDF、HTML、ePub、gitbook.ioベースのサイト、Gitレポなど。
+    - コース* ： コースは、本ではない学習教材です。[これはコースです](http://ocw.mit.edu/courses/electrical-engineering-and-computer-science/6-006-introduction-to-algorithms-fall-2011/)。
+    - インタラクティブなチュートリアル ユーザーがコードやコマンドを入力し、その結果を評価する（「評価する」というのは「採点する」という意味ではない）インタラクティブなウェブサイト： [Haskellを試す](http://tryhaskell.org)、[Gitを試す](https://learngitbranching.js.org)。
+    - Playgrounds*：プログラミング学習のためのオンラインかつインタラクティブなウェブサイト、ゲーム、またはデスクトップソフトウェアです。コードの断片を書いたり、コンパイル（または実行）したり、共有したりすることができる。プレイグラウンドでは多くの場合、フォークしてコードで遊んで手を汚すことができます。
+    - ポッドキャストとスクリーンキャスト* ： ポッドキャストとスクリーンキャスト。
+    - 問題集と競技プログラミング* ： 簡単な問題や複雑な問題を解くことで、自分のプログラミング・スキルを評価することができるウェブサイトやソフトウェア。
+
+4. 以下のガイドライン](#guidelines)を必ず守り、ファイルの[Markdownフォーマット](#formatting)を尊重してください。
```
Comment (whitehara) [inline docs/CONTRIBUTING-jp.md]: "[" is missed before "ガイドライン".

File: docs/HOWTO-jp.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+[![EbookFoundation/free-programming-books's Contributor over time Graph](https://contributor-overtime-api.apiseven.com/contributors-svg?chart=contributorOverTime&repo=ebookfoundation/free-programming-books)](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=ebookfoundation/free-programming-books)
+
+[![EbookFoundation/free-programming-books's Monthly Active Contributors graph](https://contributor-overtime-api.apiseven.com/contributors-svg?chart=contributorMonthlyActivity&repo=ebookfoundation/free-programming-books)](https://www.apiseven.com/en/contributor-graph?chart=contributorMonthlyActivity&repo=ebookfoundation/free-programming-books)
+
+</details>
+
+経験豊富なオープンソース貢献者であっても、つまずくことがあるかもしれません。PRを投稿すると、***GitHub Actions*が**リンター**を実行し、しばしば間隔やアルファベット表記に関する小さな問題を見つけます**。緑色のボタンが表示されれば、レビューの準備は完了です。もし表示されない場合は、失敗したチェックの下にある「詳細」をクリックして、リンターが何を気に入らなかったのかを調べ、PR をオープンしたブランチに新しいコミットを追加して問題を修正しましょう。
+
+最後に、もしあなたが追加したいリソースが `Free-Programming-Books` にふさわしいかどうか確信が持てない場合は、[CONTRIBUTING](CONTRIBUTING.md) *([translations](README.md#translations)もあります)* にあるガイドラインに目を通してください。
```
Comment (whitehara) [inline docs/HOWTO-jp.md]: I suggest to change "CONTRIBUTING.md" to "CONTRIBUTING-ja.md".

File: docs/CONTRIBUTING-jp.md (hunk 5 above)
Comment (aneeshd27) [reply docs/CONTRIBUTING-jp.md]: I have made the changes

File: docs/CONTRIBUTING-jp.md (hunk 6 above)
Comment (aneeshd27) [reply docs/CONTRIBUTING-jp.md]: I have made the changes

File: docs/CONTRIBUTING-jp.md [hunk 8]
```diff
@@ -0,0 +1,271 @@
...
+    マークダウン
+    * 翻訳本](http://example.com/book.html) - John Doe, `trl.:` Mike The Translator
+    ```
+
+    この例では、`trl.:`という注釈にMARCのリレータコードを使用しています。
+- コンマ`,`で著者リストの各項目を区切ります。
+- 著者リストは"`et al.`"で短縮できます。
+- クリエイターへのリンクは許可しません。
+- コンピレーションやリミックス作品の場合、"creator "には説明が必要な場合があります。例えば、"GoalKicker "や "RIP Tutorial "の本は、"`Compiled from StackOverflow documentation`"とクレジットされます。
+- Prof.」や「Dr.」のような敬称はクリエイター名に含めません。
```
[the remark this answers is not in the document]
Comment (aneeshd27) [reply docs/CONTRIBUTING-jp.md]: Made necessary changes

File: docs/HOWTO-jp.md (hunk 7 above)
Comment (aneeshd27) [reply docs/HOWTO-jp.md]: Renamed CONTRIBUTING.md to CONTRIBUTING-ja.md

**Comment by eshellman** [comment]
Looks good. Please also add links in docs/README.md

**Comment by eshellman** [comment]
In the future, to avoid conflicts, periodically update from main.

**Pull request merged**